project/plugins.sbt

```diff
@@ -1,3 +1,3 @@
 addSbtPlugin("com.eed3si9n" % "sbt-buildinfo" % "0.7.0")
 addSbtPlugin(
-  "com.47deg" % "sbt-org-policies" % "0.7.4" exclude ("io.get-coursier", "sbt-coursier"))
+  "com.47deg" % "sbt-org-policies" % "0.7.5" exclude ("io.get-coursier", "sbt-coursier"))
```

Comment:
@juanpedromoreno is this all that needed or do I still need to fiddle with .travis.yml

Reply:
Mmm, I'm not fully sure. @suhasgaddam Does 0.7.5 version fix the realese issue? Thanks.

Reply:
Yes, `0.7.5` fixes `orgAfterCISuccess` for all normal builds.